client/src/pages/CreateOrganizationProfile.js

```diff
@@ -0,0 +1,325 @@
+        Create Organization Profile
+      </Heading>
+      <StyledForm onSubmit={handleSubmit}>
+        <InputWrapper>
+          <Label style={StyleLabel} label="Organization Name" />
+          <Input
+            type="text"
+            required
+            placeholder="notion"
+            value={name}
+            onChange={handleInputChangeName}
+            style={StyleInput}
+          />
+        </InputWrapper>
+        <WhiteSpace />
+        <WhiteSpace />
+        <InputWrapper>
+          <Label style={StyleLabel} label="Organization Contact Email" />
+          <Input
+            type="email"
+            required
+            placeholder="help@notion.com (this is a fake)"
+            value={email}
+            onChange={handleInputChangeEmail}
+            style={StyleInput}
+          />
+        </InputWrapper>
+        <WhiteSpace />
+        <WhiteSpace />
+        <InputWrapper>
+          <Label style={StyleLabel} label="Location" />
+          <Input
+            type="text"
+            required
+            placeholder="Pittsburgh, Pennsylvania, United States"
+            value={location}
+            onChange={handleInputChangeLocation}
+            style={StyleInput}
+          />
+        </InputWrapper>
+        <WhiteSpace />
+        <WhiteSpace />
+        <InputWrapper>
+          <StyledCheckbox value="global" onChange={handleInputChangeGlobal}>
+            We are a global organization
+          </StyledCheckbox>
+        </InputWrapper>
+        <WhiteSpace />
+        <WhiteSpace />
+        <InputWrapper>
+          <Label style={StyleLabel} label="Type and Industry" />
+        </InputWrapper>
+        <div className="settings">
+          <CustomModal
+            title={selected ? createOrganizationProfile[selected].title : ""}
+            className="post-modal"
+            content={
+              <RadioGroup
+                flex={true}
+                padding="1.3rem 0"
+                onChange={handleFormData(selected)}
+                options={options}
+                value={formData[selected]}
+                defaultValue={formData[selected]}
+              />
+            }
+            onClose={closeModal}
+            visible={modal}
+            closable={false}
+          />
+          <div className="buttons">
+            <DownArrowButton
+              handleClick={showModal(type)}
+              label={formData.type}
+              color={theme.colors.royalBlue}
+              bgcolor="#fff"
+              long="true"
+            />
+            <WhiteSpace />
+            <DownArrowButton
+              handleClick={showModal(industry)}
+              label={formData.industry}
+              color={theme.colors.royalBlue}
+              bgcolor="#fff"
+              long="true"
+            />
+          </div>
+
+          <InputWrapper>
+            <Label style={StyleLabel} label="What are you looking for" />
+            <StyledCheckbox
```

Comment:
For the checkboxes since the only thing different is the text, maybe we can map through an array that contains the text values and render the JSX like that? And then  maybe we could create a dynamic handler to handle all the input state?

For now what we have is fine but wanted to bring that up anyways.

Reply:
This will be fixed by #399

Reply:
This will be fixed by #399